try:
    some_scary_function()
except:
    print("An error occurred. Moving on!")
